app/models/repair_params.rb

```diff
@@ -4,9 +4,9 @@ def initialize(answers)
   end
 
   def problem
-    @answers.fetch('describe_repair').fetch('description').tap do |problem|
-      return 'n/a' if problem.blank?
-    end
+    return 'n/a' if description.blank?
```

Comment:
Won't this cause the room information to be lost when description is blank..?

Reply:
Good catch! I'll add a spec